openshift/e2e-common.sh

```diff
@@ -192,11 +192,7 @@ EOF
   path: spec.install.spec.deployments.(name==knative-operator).spec.template.spec.volumes[+]
   value:
     name: "kourier-manifest"
```

Comment:
nit: Should we choose a different name, to not confuse peeps?

Reply:
Sure, updated.